app/routes/receive_package.js

```diff
@@ -14,8 +13,20 @@ export default AuthorizeRoute.extend({
     controller.set("autoGenerateInventory", true);
     controller.set("inputInventory", false);
     model.get("inventoryNumber");
+    if (model.get("isReceived")) {
+      window.history.back();
```

Comment:
Add return statement after this line so rest of the code won't be executed.

Reply:
added